README.md

```diff
@@ -4,11 +4,11 @@
 
 Some guy said:
 
-> How great would it be if we could use well maintained [Postman](https://www.getpostman.com/) collections to generate beautiful documentations with a single command?
+> How great would it be if we could use the well maintained [Postman](https://www.getpostman.com/) collections to generate beautiful documentations with a single command?
```

Comment:
Pas sûr à 100%... mais à 99% 😊 après si tu as le courage de lire jusqu'au bout je pense qu'il y a mieux à faire, mais sans le the c'est grammaticalement incorrect. Si ça t'intéresse voici les règles sur le "the".
**1er point :** Pour les articles indéfinis (un/des) en anglais "des" n'existe effectivement pas au pluriel a cat -> cats. Mais il s'agit il me semble ici d'un article défini (le/les) the cat -> the cats car tu vise des collections bien précises. C'est une erreur très courante en anglais, mais je pense que tu connais bien cette règle, c'est juste pour éclaircir les choses au cas où il y ait confusion.
**2ème point :** je pense que tu pensais en cette règle en me disant qu'on évitait "the" au pluriel.
Alors pareil c'est une erreur commune, l'anglais est très précis à ce sujet, on ne met pas des fois "the", des fois rien. On doit mettre the dans tous les pluriels à référence spécifique, et on ne met jamais the dans tous les pluriels à référence générales.
_Les chats dorment -> the cats are sleeping_ (tu parles de tes chats qui pioncent devant toi)
_Les chats détestent l'eau -> cats hate water_ (notion générale)
Du coup dans ton cas il faut absolument préciser "the" pour viser les collections de Postman, sinon tu viserais de manière générale les bonnes collections, sauf qu'ici tu ne peux pas : tu as précisé "Postman".
Au final on a pas le choix mais ça fait pas très joli ou je trouve que ça sonne pas anglais, peut-être qu'ils écriraient plutôt "Postman's well maintained collections"
Pour revenir aux exemple on aurait :

_Les chats détestent l'eau -> cats hate water _(notion générale) - rien ne change
_Les chats dorment -> the cats are sleeping_ (tu parles de tes chats qui pioncent devant toi), idem
_Les chats d'Alex dorment -> the cats of Alex are sleeping_ -> correct mais moche -> on renverse _-> Alex's cats are sleeping._ Du coup on évite le "the" au pluriel en mettant ce possessif, c'est si courant qu'entre ça et le cas générique on voit finalement peu ce "the" au pluriel, mais ça existe bien ! _Children are crazy about smartphones. The children are starving._